eks_fargate/README.md

```diff
@@ -373,3 +375,4 @@ Need help? Contact [Datadog support][20].
 [19]: http://docs.datadoghq.com/agent/cluster_agent/event_collection
 [20]: https://docs.datadoghq.com/help/
 [21]: https://docs.datadoghq.com/agent/kubernetes/daemonset_setup/?tab=k8sfile#process-collection
+[22]: https://docs.datadoghq.com/integrations/ecs_fargate
```

Comment:
Should this link to eks integration page?

Reply:
no—this is on the EKS integration page, it's supposed to be a link to the ECS integration in case people are in the wrong palce